qiita_db/test/test_archive.py

```diff
@@ -0,0 +1,68 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from json import dumps
+
+from qiita_core.util import qiita_test_checker
+import qiita_db as qdb
+
+
+@qiita_test_checker()
+class ArchiveTest(TestCase):
+    def test_insert_from_biom_and_retrieve_feature_values(self):
+        # 1 - to test error as it's FASTQ
+        with self.assertRaises(ValueError) as err:
+            qdb.archive.Archive.insert_from_artifact(
+                qdb.artifact.Artifact(1), {})
+        self.assertEqual(
+            str(err.exception), 'To archive artifact must be BIOM but FASTQ')
+
+        # 7 - to test error due to not filepath biom
```

Comment:
This is testing that an artifact of type BIOM doesn't have a BIOM? that seems like it should never happen.

Reply:
right, but I think an extra test never hurts ... anyway, want me to remove it?